content/vitest.config.ts

```diff
@@ -0,0 +1,19 @@
+import { defineConfig } from "vitest/config";
+import path from "path";
+
+export default defineConfig({
+  resolve: {
+    alias: {
+      "@businessnjgovnavigator/shared": path.resolve(__dirname, "../shared/src"),
+    },
+  },
+  test: {
+    coverage: {
```

Comment:
I know, I know. We don't use coverage elsewhere, and 100% test coverage isn't an indicator you've done testing right.

That said.

_If_ we ever turn on test coverage again to see where gaps are, we're prepared ahead of time here.